philipmeneghini/InstaConnect
# Home & Profile Page Implementation

Home page implemented for when user's login. Displays users that they are following posts and enables them to like or comment on them.

![HomePage](https://github.com/philipmeneghini/InstaConnect/assets/63440066/d0aca496-5654-42b2-9b29-f02ab4ace024)

**Testing**

- Check to make sure that both like and comment functionality properly populates the database
- Make sure the interface looks good on multiple screen sizes
- Check that all content is properly displayed in profile page and home page for followers of who logged in
